components/schema-migrator/migrations/director/20211015154700_unpairing-application.down.sql

```diff
@@ -0,0 +1,84 @@
+BEGIN;
+
+DROP VIEW IF EXISTS tenants_apps;
+
+ALTER TABLE applications
+    ALTER COLUMN status_condition DROP DEFAULT;
+
+ALTER TYPE application_status_condition RENAME TO application_status_condition_old;
+CREATE TYPE application_status_condition AS ENUM (
+    'INITIAL',
+    'CONNECTED',
+    'FAILED',
+    'CREATING',
+    'CREATE_FAILED',
+    'CREATE_SUCCEEDED',
+    'UPDATING',
+    'UPDATE_FAILED',
+    'UPDATE_SUCCEEDED',
+    'DELETING',
+    'DELETE_FAILED',
+    'DELETE_SUCCEEDED',
+    'MANAGED'
+);
+
+ALTER TABLE applications
+    ALTER COLUMN status_condition TYPE application_status_condition
+        USING status_condition::text::application_status_condition;
+
+ALTER TABLE applications
+    ALTER COLUMN status_condition
+        SET DEFAULT 'INITIAL' ::application_status_condition;
+
+DROP TYPE application_status_condition_old;
+
+CREATE OR REPLACE VIEW tenants_apps
+            (tenant_id, provider_tenant_id, id, name, description, status_condition, status_timestamp, healthcheck_url,
```

Comment:
add a comment around the actual change in the view so that its easier for review